layers/+tools/restclient/packages.el

```diff
@@ -35,8 +38,29 @@
       (unless restclient-use-org
         (add-to-list 'auto-mode-alist '("\\.http\\'" . restclient-mode)))
       (spacemacs/set-leader-keys-for-major-mode 'restclient-mode
+        "n" 'restclient-jump-next
```

Comment:
I think it would be better to have it under `SPC g`. So next is `SPC g n` and previous is `SPC g p`. What do you think?